README.md

```diff
@@ -28,3 +28,27 @@ jobs:
 |PLF_END_POINT_URL| The URL of your pipeline factory instance API|
 |PLF_API_KEY| A valid pipeline factory API Key. Store in Github Secrets |
 
+## Repository Settings File
+
+Add  pipeline-factory.settings file to the root of your repository to override default settings.
+- file format is JSON
+- This file needs to be in repository root path to be considered.
+- File name is case-sensitive.
+- file is optional 
+- All file keys are optional. 
+
+```json
+
+{
+  // name of s3 bucket to store build artifacts
+  "artifacts_bucket_name" : "s3 bucket name" ,
+
+  // iam role to use for , 
+  "build_as_role_arn" : "iam role arn",
+
+  // relative location of build spec file , if omitted default is "buildspec.yml"
+  "buildspec_file_location" : "./scripts/custom_buildspec.yml",
```

Comment:
The casing on these settings is inconsistent for JSON configs. I would expect to see camelCase normally for JSON configs (see TS config / Jest config etc)

Reply:
Done